assignment01/index.html

```diff
@@ -0,0 +1,33 @@
+<!DOCTYPE html>
+<html lang="en">
+
+<head></head>
+
+<body>
+<!-- James Welu 
+assignment01.htm
+INFO2340.WW
+Thoendel
+12/05/2019
+-->
+    <!--
+    <script src="https://ajax.googleapis.com/ajax/libs/jquery/3.4.1/jquery.min.js"></script>
+    -->
+    <script src="js/jquery-3.4.1.min.js"></script>
+    <script src="js/myScript1.js"></script>
+
+<h1>Greetings</h1>
+
+<p>Hello, my name is <span id="first-name">James</span><span id="last-name"> Welu</span>!</p>

```

Comment:
The goal of this assignment was to have an html file _exactly_ like the one in figure 1. So there should not be values for your name here, style for the mcc class below or the list items below that. All that content and style should be added using your custom js file.

Reply:
I've got it now. I rewrote the .js and accomplished what was needed to display the page correctly. It's actually included on my last "pull request" I made to try and figure out how things are supposed to be turned in. I should be ready for the next unit.